<!doctype html>
<html class="no-js" lang="en-us">
  <head>
	<meta name="generator" content="Hugo 0.92.0" />
    <meta charset="utf-8">
    <title>Computational Biology / Biological Computing</title>
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    <link rel="stylesheet" href="https://computingbiology.github.io/css/foundation.min.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/highlight.min.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/font-awesome.min.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/academicons.min.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/fonts.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/finite.css">
    <link rel="shortcut icon" href="/images/dna.png">

    <script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=TeX-MML-AM_CHTML">
</script>

    
  </head>
  <body>
      
    <header>
      <nav class="nav-bar">
	
	  <div class="title-bar" data-responsive-toggle="site-menu" data-hide-for="medium">	      
	    <button class="site-hamburger" type="button" data-toggle>
	      <i class="fa fa-bars fa-lg" aria-hidden="true"></i>
	    </button>
	    <div class="title-bar-title site-title">
	      <a href="https://computingbiology.github.io/">Computational Biology / Biological Computing</a>
	    </div>
	    <div class="title-bar-right pull-right">
	      
	      
	    </div>
	  </div>
	    
	  
	    <div class="top-bar" id="site-menu" >	      
	      <div class="top-bar-title show-for-medium site-title">
		<a href="https://computingbiology.github.io/">Computational Biology / Biological Computing</a>
	      </div>
	      <div class="top-bar-left">
		<ul class="menu vertical medium-horizontal">
		  
		  
		</ul>
	      </div>
	      <div class="top-bar-right show-for-medium">
		
	         <p class="groupstyle">Computer Science</br>University of Virginia</p>
		
	      </div>
	    </div>
	  
	</nav>
      
    </header>
    
    <main>
      





   <div class="container">       
   <div class="sidebar">
   <p>
University of Virginia<Br>
cs4501 Spring 2021<br>
<em>Computational Biology / Biological Computing</em>
</p>

   <p>
   
     <b><a href="/syllabus">Syllabus</a></b></br>
     <b><a href="/schedule">Schedule</a></b></br>

   <p></p>
<a href="https://computingbiology.slack.com/">Slack</a><br>
<a href="https://github.com/computingbiology/spring2022/discussions">Discussions</a></br>
   <p>
   </p>
   <p>
   <a href="/project2"><b>Project 2</b></a><br>
   <a href="/project1">Project 1</a>
   </p>
<p></p>
   <p>
   <b><a href="/post/">Recent Posts</a></b>
   
   <div class="posttitle">
      <a href="/class11/">Class 11: Editing Genomes</a>


   </div>
   
   <div class="posttitle">
      <a href="/class10/">Class 10: Genomes and Phenotypes</a>


   </div>
   
   <div class="posttitle">
      <a href="/class9/">Class 9: Neighbor-Joining Phylogeny</a>


   </div>
   
   <div class="posttitle">
      <a href="/class8/">Class 8: Distance-Based Phylogeny</a>


   </div>
   
   <div class="posttitle">
      <a href="/class7/">Class 7: Phylogeny</a>


   </div>
   
   <div class="posttitle">
      <a href="/class6/">Class 6: More Alignment</a>


   </div>
   
   <div class="posttitle">
      <a href="/class5/">Class 5: Sequence Alignment</a>


   </div>
   
   <div class="posttitle">
      <a href="/class4/">Class 4: Analyzing Genomes</a>


   </div>
   
   <div class="posttitle">
      <a href="/class3/">Class 3: Assembly, Human Genome Project</a>


   </div>
   
   <div class="posttitle">
      <a href="/class2/">Class 2: Genomes</a>


   </div>
   
   <div class="posttitle">
      <a href="/week1/">Class 1: Life</a>


   </div>
   
   <div class="posttitle">
      <a href="/survey/">Course Registration Survey</a>


   </div>
   
   <div class="posttitle">
      <a href="/slack/">Course Slack</a>


   </div>
   
   <div class="posttitle">
      <a href="/computer-science-background/">Computer Science Background</a>


   </div>
   
   <div class="posttitle">
      <a href="/welcome/">Welcome</a>


   </div>
   
   
   </p>

   <p><br></br><br></br><br></br><br></br><br></br><br></br></p>
      <p>
     Teacher: <a href="https://www.cs.virginia.edu/evans">David Evans</a>
   </p>

   </div>


    <div class="content">

    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    


    
    
    <h1><a href="/class11/">Class 11: Editing Genomes</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-02-23 00:00:00 &#43;0000 UTC" itemprop="datePublished">23 February 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <h2 id="schedule-and-readings">Schedule and Readings</h2>
<p><strong>Thursday, 24 Feb</strong>:</p>
<ul>
<li><a href="/project2"><strong>Project 2</strong></a></li>
</ul>
<p>New readings for next week will be posted by Friday.</p>
<h2 id="slides">Slides</h2>
<p>The slides from the 21 February class are here: <a href="https://www.dropbox.com/s/px2yhuv9b70cguq/csbio-class11.pdf?dl=0">class11.pdf</a></p>
<h2 id="links">Links</h2>
<ul>
<li>
<p>Wladyslaw Altermann, JÃ³zef Kazmierczak. <a href="/docs/altermann2003.pdf"><em>Archean microfossils: a reappraisal of early life on Earth</em></a> (Describes fossils of life from over 3.4 Billion years ago.)</p>
</li>
<li>
<p>Anja Spang, Courtney W. Stairs, Nina Dombrowski, Laura Eme, Jonathan Lombard, Eva F. Caceres, Chris Greening, Brett J. Baker and Thijs J. G. Ettema. <a href="/docs/spang2019.pdf"><em>Proposal of the reverse flow model for the origin of the eukaryotic cell based on comparative analyses of Asgard archaeal metabolism</em></a>. Nature Microbiology, July 2019.</p>
</li>
<li>
<p><a href="https://www.ams.usda.gov/rules-regulations/be">Bioengineered Food Disclosure</a></p>
</li>
<li>
<p>The best recorded lectures I found on Recombinant DNA are MIT&rsquo;s 7.01 course (Eric Lander), starting with <a href="https://www.youtube.com/watch?v=uDXH6Uu0ghc"><em>Overview of Recombinant DNA, excerpt 1 | MIT 7.01SC Fundamentals of Biology</em></a>.</p>
</li>
<li>
<p>Pavan Umate, Narendra Tuteja and Renu Tuteja. <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3073292/pdf/cib0401_0118.pdf"><em>Genome-wide comprehensive analysis of human helicases</em></a>. Communicative and Integrative Biology, 2011.</p>
</li>
<li>
<p>Patent for recombinant DNA: <a href="https://patents.google.com/patent/US4237224A/"><em>Process for Producing Biologically Functional Molecular Chimeras</em></a> (Stanley Cohen and Herbert Boyer, 1974). Rajendra Bera, <a href="/docs/cohenboyerpatents.pdf"><em>The story of the Cohen-Boyer patents</em></a>, Current Science, March 2009.</p>
</li>
<li>
<p>Paul Berg, David Baltimore, Herbert W. Boyer, Stanley N. Cohen, Ronald W. Davis, David S. Hogness, Daniel Nathans, Richard Roblin, James D. Watson, Sherman Weissman, Norton D. Zinder. <a href="/docs/berg-biohazards1974.pdf"><em>Potential Biohazards of Recombinant DNA Molecules</em></a>. Science, 1974.</p>
</li>
<li>
<p><a href="https://futureoflife.org/2017/08/11/ai-principles/">Asilomar AI Principles</a> (2017).</p>
</li>
</ul>

      </div>
<hr class="post-separator"></hr>

    
    <h1><a href="/class10/">Class 10: Genomes and Phenotypes</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-02-21 00:00:00 &#43;0000 UTC" itemprop="datePublished">21 February 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <h2 id="schedule-and-readings">Schedule and Readings</h2>
<p><strong>Thursday, 24 Feb</strong>:</p>
<ul>
<li><a href="/project2"><strong>Project 2</strong></a></li>
</ul>
<h2 id="slides">Slides</h2>
<p>The slides from the 21 February class are here: <a href="https://www.dropbox.com/s/hywo3lj2k23caao/csbio-class10.pdf?dl=0">class10.pdf</a></p>
<h2 id="links">Links</h2>
<ul>
<li>
<p>Charles Darwin. <a href="https://www.google.com/books/edition/The_Variation_of_Animals_and_Plants_Unde/hoo5AAAAcAAJ?hl=en&amp;gbpv=1"><em>The Variation of Animals and Plants Under Domestication</em></a>. 1868. (Darwin&rsquo;s strange (to us today) theory on pangenesis is Chapter XXVII, starting on page 357.)</p>
</li>
<li>
<p>Gregor Mendel. <a href="https://www.biodiversitylibrary.org/page/48299076#page/133/mode/1up"><em>Versuche uber Pflanzen-Hybriden</em></a>. Proceedings of Natural History Society, 1866 <a href="http://www.esp.org/foundations/genetics/classical/gm-65.pdf">English Translation</a></p>
</li>
<li>
<p>R. A. Fisher. <a href="/docs/fisher1936.pdf"><em>Has Mendel&rsquo;s Work Been Rediscovered?</em></a>, 15 April 1936. (This isn&rsquo;t the paper we talked about in class, but is an interesting later one.)</p>
</li>
<li>
<p>Matti Pirinen&rsquo;s GWAS course: <a href="https://www.mv.helsinki.fi/home/mjxpirin/GWAS_course/">Genome-wide Association Studies</a>, University of Helsinki.</p>
</li>
<li>
<p>Robert J. Klein, Caroline Zeiss, Emily Y. Chew, Jen-Yue Tsai, Richard S. Sackler, Chad Haynes, Alice K. Henning, John Paul SanGiovanni, Shrikant M. Mane, Susan T. Mayne, Michael B. Bracken, Frederick L. Ferris, Jurg Ott, Colin Barnstable, Josephine Hoh. <a href="/docs/klein2005.pdf"><em>Complement Factor H Polymorphism in Age-Related Macular Degeneration</em></a>. Science, 15 April 2005.</p>
</li>
<li>
<p>James J. Lee, et al. <a href="/docs/gwaseducation.pdf"><em>Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals</em></a>. Nature Genetics, August 2018.</p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/gwas/diagram">GWAS Catalog</a></p>
</li>
</ul>

      </div>
<hr class="post-separator"></hr>

    
    <h1><a href="/class9/">Class 9: Neighbor-Joining Phylogeny</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-02-16 00:00:00 &#43;0000 UTC" itemprop="datePublished">16 February 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <h2 id="schedule-and-readings">Schedule and Readings</h2>
<p>See <a href="/class8">Class 8</a>.</p>
<h2 id="slides">Slides</h2>
<p>The slides from the 16 February class are here: <a href="https://www.dropbox.com/s/r6h0flm8jo7vkt5/csbio-class9.pdf?dl=0">class9.pdf</a></p>
<h2 id="links">Links</h2>
<ul>
<li>
<p>J. Marmurt and P. Doty. <a href="/docs/marmur1962.pdf"><em>Determination of the Base Composition of
Deoxyribonucleic Acid from its Thermal Denaturation
Temperature</em></a>. Journal of Molecular Biology, 1962.</p>
</li>
<li>
<p><a href="https://www.cs.virginia.edu/~evans/paradise/teaching/#grading"><em>How to Grade</em></a> (this is a write-up of my grading system, although the symbols are slightly different from what we are using in this class). Since we are using &ldquo;virtual&rdquo; goal stars, you don&rsquo;t need to be worried about <a href="https://www.theonion.com/bush-earmarks-1-5-billion-gold-stars-for-education-1819566310">any alarming gold star shortages</a>.</p>
</li>
<li>
<p>Maruya Saitou and Masatoshi Nei. <a href="/docs/saitou1987.pdf"><em>The Neighbor-joining Method: A New Method for Reconstructing Phylogenetic Trees</em></a>. Molecular Biological Evolution, 1987.</p>
</li>
<li>
<p>Letter organized by Peter Daszak, <a href="https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30418-9/fulltext"><em>Statement in support of the scientists, public health professionals, and medical professionals of China combatting COVID-19</em></a>, The Lancet, 19 February 2020. <a href="https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(21)01377-5/fulltext">Addendum on competing interests</a>, 21 June 2021.</p>
</li>
<li>
<p>Kristian G. Andersen, Andrew Rambaut, W. Ian Lipkin, Edward C. Holmes and Robert F. Garry . <a href="https://www.nature.com/articles/s41591-020-0820-9"><em>The proximal origin of SARS-CoV-2</em></a>. Nature Medicine, 17 March 2020.</p>
</li>
<li>
<p>There are hundreds of articles with different viewpoints on the more likely origins of Covid-19. These are two of the more balanced ones:</p>
<ul>
<li>
<p>Carolyn Kormann. <a href="https://www.newyorker.com/science/elements/the-mysterious-case-of-the-covid-19-lab-leak-theory"><em>The Mysterious Case of the COVID-19 Lab-Leak Theory</em></a>. The New Yorker, 12 October 2021.</p>
</li>
<li>
<p>Claire Felter. <a href="https://www.cfr.org/backgrounder/will-world-ever-solve-mystery-covid-19s-origin#chapter-title-0-5"><em>Will the World Ever Solve the Mystery of COVID-19âs Origin?</em></a>. Council on Foreign Relations, 3 November 2021.</p>
</li>
</ul>
</li>
<li>
<p>Alina Chan and Matt Ridley&rsquo;s book <a href="https://www.amazon.com/Viral-Search-COVID-19-Matt-Ridley/dp/006313912X"><em>Viral: The Search for the Origin of COVID-19</em></a> pretends to be balanced (and I believe is mostly solid), but has a clear agenda. <a href="https://scottaaronson.blog/?p=6183">Scott Aaronson&rsquo;s (positive) Review</a>, <a href="https://newrepublic.com/article/164688/viral-lab-leak-theory-covid-19">New Republic review (take down)</a>.</p>
</li>
</ul>

      </div>
<hr class="post-separator"></hr>

    
    <h1><a href="/class8/">Class 8: Distance-Based Phylogeny</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-02-14 00:00:00 &#43;0000 UTC" itemprop="datePublished">14 February 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <h2 id="schedule-and-readings">Schedule and Readings</h2>
<p>(These are duplicated from <a href="/class7">Class 7</a>.)</p>
<p>Before <strong>Wednesday, 9 February</strong>:</p>
<ul>
<li>Read Chapter 19 &ndash; 26 of Hubert&rsquo;s <a href="https://berthub.eu/dna-book/toc-real/">The DNA
Book</a>. (Chapter 26 on <em>Mutations and Changes</em> is the one most relevant to today&rsquo;s class.)</li>
<li>Read Chapter 5 of the <em>Bioinformatics
Algorithms</em>: <a href="//www.bioinformaticsalgorithms.org/bioinformatics-chapter-5"><em>How do we compare biological sequences?</em></a>.</li>
</ul>
<p>Before <strong>Monday, 14 Feb</strong>:</p>
<ul>
<li>Chapter 26 of  Manolis Kellis et al., <em>Computational Biology - Genomes, Networks, and Evolution</em>: <a href="https://bio.libretexts.org/Bookshelves/Computational_Biology/Book%3A_Computational_Biology_-_Genomes_Networks_and_Evolution_(Kellis_et_al.)/26%3A_Molecular_Evolution_and_Phylogenetics"><em>Molecular Evolution and Phylogenetics</em></a> (also available as [<a href="/docs/kellis-ch26.pdf">PDF</a>]) (if you prefer to watch a video lecture that this chapter is based on, you can watch <a href="https://www.youtube.com/watch?v=_KduKR6IXcA"><em>MIT CompBio Lecture 19 - Phylogenetics</em></a>).</li>
</ul>
<p><strong>Thursday, 24 Feb</strong>:</p>
<ul>
<li><a href="/project2"><strong>Project 2</strong></a></li>
</ul>
<h2 id="slides">Slides</h2>
<p>The slides from the 14 February class are here: <a href="https://www.dropbox.com/s/yf8iu3edh6avjrn/csbio-class8-inked.pdf?dl=0">class8.pdf</a></p>
<h2 id="links">Links</h2>
<ul>
<li>
<p>The graph on Covid virus samples is from: <a href="https://www.mwra.com/biobot/biobotdata.htm">Massachusetts Water Resrouces Authority Wastewater COVID-19 Tracking</a>.</p>
</li>
<li>
<p>Mariana Guadalupe Matus Garcia. <a href="http://dspace.mit.edu/handle/1721.1/119603"><em>Analysis of fecal biomarkers to impact clinical care and public health</em></a>. MIT PhD Dissertation (Computational and Systems Biology), September 2018. (If you think PhD dissertations need to be intiminating, long, documents, please look at this one! Its only about 50 double-spaced sparse pages long, if you don&rsquo;t count the appendices, which are mostly pictures and directions of how you sample sewage.) The company that she co-founded that produced the data for the MWRA COVID-19 Tracking graph is <a href="https://www.biobot.io/">Biobot Analytics</a>.</p>
</li>
<li>
<p>John C Avise, Robert A Lansman, Rosemary O Shade. <a href="/docs/avise1979.pdf"><em>The use of Restriction Endonucleases to Measure Mitochondrial DNA Sequence Relatedness in Natural Populations</em></a>. Genetics, May 1979.</p>
</li>
<li>
<p>Joel D. Velasco. <a href="/docs/velasco2008.pdf"><em>The prior probabilities of phylogenetic
trees</em></a>. Biology and Philosophy. 208.</p>
</li>
<li>
<p>Yoshihito Niimura, Atsushi Matsui1, and Kazushige Touhara. <a href="https://genome.cshlp.org/content/24/9/1485.long"><em>Extreme expansion of the olfactory receptor gene repertoire in African elephants and evolutionary dynamics of orthologous gene groups in 13 placental mammals</em></a>. Genome Research, 22 July 2014.</p>
</li>
<li>
<p>Yoav Gilad, Victor Wiebe, Molly Przeworski, Doron Lancet, Svante PÃ¤Ã¤bo. <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0020005"><em>Loss of Olfactory Receptor Genes Coincides with the Acquisition of Full Trichromatic Vision in Primates</em></a>, PLOS Biology, January 2004. (This is the paper that establishes correlation with color vision and loss of olfactory receptor genes.)</p>
</li>
</ul>

      </div>
<hr class="post-separator"></hr>

    
    <h1><a href="/class7/">Class 7: Phylogeny</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-02-09 00:00:00 &#43;0000 UTC" itemprop="datePublished">9 February 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <h2 id="schedule-and-readings">Schedule and Readings</h2>
<p>Before <strong>Wednesday, 9 February</strong>:</p>
<ul>
<li>Read Chapter 19 &ndash; 26 of Hubert&rsquo;s <a href="https://berthub.eu/dna-book/toc-real/">The DNA
Book</a>. (Chapter 26 on <em>Mutations and Changes</em> is the one most relevant to today&rsquo;s class.)</li>
<li>Read Chapter 5 of the <em>Bioinformatics
Algorithms</em>: <a href="//www.bioinformaticsalgorithms.org/bioinformatics-chapter-5"><em>How do we compare biological sequences?</em></a>.</li>
</ul>
<p>Before <strong>Monday, 14 Feb</strong>:</p>
<ul>
<li>Chapter 26 of  Manolis Kellis et al., <em>Computational Biology - Genomes, Networks, and Evolution</em>: <a href="https://bio.libretexts.org/Bookshelves/Computational_Biology/Book%3A_Computational_Biology_-_Genomes_Networks_and_Evolution_(Kellis_et_al.)/26%3A_Molecular_Evolution_and_Phylogenetics"><em>Molecular Evolution and Phylogenetics</em></a> (also available as [<a href="/docs/kellis-ch26.pdf">PDF</a>]) (if you prefer to watch a video lecture that this chapter is based on, you can watch <a href="https://www.youtube.com/watch?v=_KduKR6IXcA"><em>MIT CompBio Lecture 19 - Phylogenetics</em></a>).</li>
</ul>
<p><strong>Thursday, 24 Feb</strong>:</p>
<ul>
<li><a href="/project2"><strong>Project 2</strong></a></li>
</ul>
<h2 id="slides">Slides</h2>
<p>The slides from the 9 February class are here: <a href="https://www.dropbox.com/s/ixdjyjgpx3ped2l/csbio-class7-inked.pdf?dl=0">class7.pdf</a></p>
<p>Some links to materials for the class:</p>
<ul>
<li>
<p>Georgia Ray (&ldquo;Eukaryote Write Blog&rdquo;), <a href="https://eukaryotewritesblog.com/2021/05/02/"><em>Thereâs no such thing as a tree (phylogenetically)</em></a>. May 2021.</p>
</li>
<li>
<p>Thomas Bayes, conveyed by Mr. Price, <a href="/docs/bayes1763.pdf"><em>As Essay towards solving a Problem in the Doctrine of Chances</em></a>, 1763. Biography of Bayes: D.R. Bellhouse, <a href="https://biostat.jhsph.edu/courses/bio621/misc/bayesbiog.pdf"><em>The Reverend Thomas Bayes, FRS:
A Biography to Celebrate the Tercentenary of His Birth</em></a>, 2004.</p>
</li>
<li>
<p>Recent New York Times articles about Bayesian reasoning: <a href="https://www.nytimes.com/2020/08/04/science/coronavirus-bayes-statistics-math.html">_How to Think Like an Epidemiologist</a>, 4 August 2020; <a href="https://www.nytimes.com/2022/01/01/upshot/pregnancy-birth-genetic-testing.html"><em>When They Warn of Rare Disorders, These Prenatal Tests Are Usually Wrong</em></a>, 1 January 2022.</p>
</li>
<li>
<p>Russell D. Gray, Simon Greenhil and Robert M. Moss. <a href="https://simon.net.nz/publication/Gray_et_al_2007.pdf"><em>The Pleasures and Perils of Darwinizing Culture (with phylogenies)</em></a>. 2007.</p>
</li>
<li>
<p>FabrÃ­cia F. Nascimento, Mario dos Reis, and Ziheng Yang. <a href="/docs/nascimento2017.pdf"><em>A
biologistâs guide to Bayesian phylogenetic analysis</em></a>. Nature Ecology and Evolution, October 2017.</p>
</li>
</ul>

      </div>
<hr class="post-separator"></hr>

    

   

    <footer>
      <nav>
	<a href="/post/" class="button hollow primary">All Posts</a>
      </nav>
    </footer>

</div>
</div>


    </main>
    
    
    <footer class="whatisthis">
  <hr />
  <div class="row">
    <div class="column small-12 medium-6">
      <a href="//computingbiology.github.io"><b>cs 4501: Computational Biology / Biological Computing</b></a><br>
      <a href="//www.cs.virginia.edu">University of Virginia</a>
    </div>
    <div class="column small-14 medium-5">
      <font size="-1">
      Subscribe to
	the <a href="/index.xml"><i class="fa fa-rss-square"></i>&nbsp;RSS feed</a>.
      <a id="searchsite">
	<form method="get" action="https://duckduckgo.com/">
	  <label for="search-field" class="show-for-sr">Search with DuckDuckGo</label>
	  <input type="search" name="q" maxlength="255" placeholder="Search with DuckDuckGo" id="search-field">
	  <input type="hidden" name="sites" value="https://computingbiology.github.io/"/>
	  <input type="hidden" name="k7" value="#faf8f8"/>
	  <input type="hidden" name="kj" value="#b33"/>
	  <input type="hidden" name="ky" value="#fafafa"/>
	  <input type="hidden" name="kx" value="b"/>
	  <input type="hidden" name="ko" value="-1"/>
	  <input type="hidden" name="k1" value="-1"/>
	  <input type="submit" value="DuckDuckGo Search" style="visibility: hidden;" />
	</form>
      </a>
</font>
    </div>
  </div>
  </hr>

</footer>


    
    
    <div class="endofpage">
    </div>

    <script src="/js/jquery.js"></script>
    <script src="/js/what-input.js"></script>
    <script src="/js/foundation.min.js"></script>
    <script src="/js/finite.js"></script>

    
    <script src="/js/highlight.pack.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>

    
    
    
  </body>
</html>
