<!doctype html>
<html class="no-js" lang="en-us">
  <head>
	<meta name="generator" content="Hugo 0.71.0" />
    <meta charset="utf-8">
    <title>Computational Biology / Biological Computing</title>
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    <link rel="stylesheet" href="https://computingbiology.github.io/css/foundation.min.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/highlight.min.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/font-awesome.min.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/academicons.min.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/fonts.css">
    <link rel="stylesheet" href="https://computingbiology.github.io/css/finite.css">
    <link rel="shortcut icon" href="/images/dna.png">

    <script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=TeX-MML-AM_CHTML">
</script>

    
  </head>
  <body>
      
    <header>
      <nav class="nav-bar">
	
	  <div class="title-bar" data-responsive-toggle="site-menu" data-hide-for="medium">	      
	    <button class="site-hamburger" type="button" data-toggle>
	      <i class="fa fa-bars fa-lg" aria-hidden="true"></i>
	    </button>
	    <div class="title-bar-title site-title">
	      <a href="https://computingbiology.github.io/">Computational Biology / Biological Computing</a>
	    </div>
	    <div class="title-bar-right pull-right">
	      
	      
	    </div>
	  </div>
	    
	  
	    <div class="top-bar" id="site-menu" >	      
	      <div class="top-bar-title show-for-medium site-title">
		<a href="https://computingbiology.github.io/">Computational Biology / Biological Computing</a>
	      </div>
	      <div class="top-bar-left">
		<ul class="menu vertical medium-horizontal">
		  
		  
		</ul>
	      </div>
	      <div class="top-bar-right show-for-medium">
		
	         <p class="groupstyle">Computer Science</br>University of Virginia</p>
		
	      </div>
	    </div>
	  
	</nav>
      
    </header>
    
    <main>
      





   <div class="container">       
   <div class="sidebar">
   <p>
University of Virginia<Br>
cs4501 Spring 2021<br>
<em>Computational Biology / Biological Computing</em>
</p>

   <p>
   
     <b><a href="/syllabus">Syllabus</a></b></br>
     <b><a href="/schedule">Schedule</a></b></br>

   <p></p>
<a href="https://computingbiology.slack.com/">Slack</a><br>
<a href="https://github.com/computingbiology/spring2022/discussions">Discussions</a></br>
   <p>
   </p>
   <p>
   <a href="/project2"><b>Project 2</b></a><br>
   <a href="/project1">Project 1</a>
   </p>
<p></p>
   <p>
   <b><a href="/post/">Recent Posts</a></b>
   
   <div class="posttitle">
      <a href="/finalproject/">Final Project</a>


   </div>
   
   <div class="posttitle">
      <a href="/crisprpatent/">Update: CRISPR Patent Ruling</a>


   </div>
   
   <div class="posttitle">
      <a href="/class12/">Class 12: CRISPR in Bacteria</a>


   </div>
   
   <div class="posttitle">
      <a href="/class11/">Class 11: Editing Genomes</a>


   </div>
   
   <div class="posttitle">
      <a href="/class10/">Class 10: Genomes and Phenotypes</a>


   </div>
   
   <div class="posttitle">
      <a href="/class9/">Class 9: Neighbor-Joining Phylogeny</a>


   </div>
   
   <div class="posttitle">
      <a href="/class8/">Class 8: Distance-Based Phylogeny</a>


   </div>
   
   <div class="posttitle">
      <a href="/class7/">Class 7: Phylogeny</a>


   </div>
   
   <div class="posttitle">
      <a href="/class6/">Class 6: More Alignment</a>


   </div>
   
   <div class="posttitle">
      <a href="/class5/">Class 5: Sequence Alignment</a>


   </div>
   
   <div class="posttitle">
      <a href="/class4/">Class 4: Analyzing Genomes</a>


   </div>
   
   <div class="posttitle">
      <a href="/class3/">Class 3: Assembly, Human Genome Project</a>


   </div>
   
   <div class="posttitle">
      <a href="/class2/">Class 2: Genomes</a>


   </div>
   
   <div class="posttitle">
      <a href="/class1/">Class 1: Life</a>


   </div>
   
   <div class="posttitle">
      <a href="/survey/">Course Registration Survey</a>


   </div>
   
   
   </p>

   <p><br></br><br></br><br></br><br></br><br></br><br></br></p>
      <p>
     Teacher: <a href="https://www.cs.virginia.edu/evans">David Evans</a>
   </p>

   </div>


    <div class="content">

    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    


    
    
    <h1><a href="/finalproject/">Final Project</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-03-02 00:00:00 &#43;0000 UTC" itemprop="datePublished">2 March 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <p>As discussed in <a href="/https://computingbiology.github.io/class4/">Class
4</a>, your final project
can be on anything you want that is related to this course. We want
your final projects to be interesting and fun to work on, and to
produce something of value beyond just satisfying an expectation of
this course.</p>
<p>We are expecting most students will do one of these types of projects:</p>
<ul>
<li>
<p><strong>Systematization Projects</strong>: Start with a curious question, learn
what is currently understood about it from the research literature, and produce something to explain
it to computer scientists. This could be a static or interactive web page, a video, improving a Wikipedia page, or some other artifact.</p>
</li>
<li>
<p><strong>Research Project</strong>: conduct original research on a relevant
topic. This starts by understanding what is already known and
identifying a question to study. Then, applying tools you learned
in this class, and others you learn on your own, to make progress
on answering that question.</p>
</li>
<li>
<p><strong>Tool-Building/Improving Project</strong>: develop some tool that will be
useful for computational biology research. It is probably more
useful to make improvements or add a feature to a widely-used open
source tool, than to build your own software from scratch.</p>
</li>
<li>
<p><strong>Hobby Project</strong>: replicate something for fun (e.g., sequence your
genome), and produce an artifact (most likely a web site) explaining
your experience in a way that will be interesting and useful to
others.</p>
</li>
</ul>
<p>Your project doesn&rsquo;t need to fit into any of these categories, but
these should give you some ideas for the kinds of project you might
do.</p>
<h2 id="project-goals">Project Goals</h2>
<p>An ideal project will satisfy these desiderata:</p>
<ul>
<li>fun (for you to do, and for others to see)</li>
<li>useful (at least to yourself, but hopefully to many)</li>
<li>relevant (to the class, to yourself, to humanity)</li>
<li>technically interesting</li>
</ul>
<p>Your project is not required to satisfy <em>all</em> of these, but should
satisfy at least three of them, and satisfy at least one of them
especially well.</p>
<h2 id="teams">Teams</h2>
<p>For the final project, you may work alone or in a team of your
(mutual) choosing with any number of people. The impressiveness of
your project should scale as at least the square root of the number of
people on your team (so, for example, a team of three people would be
expected to do something approximately 1.7 times as impressive as a
one-person project).</p>
<p>You should not underestimate the need for management and structure if
you have a large (this means more than one person!) team. If you have
more than one person on your team, your project proposal should
include a plan for how to manage your team and work well together.</p>
<h2 id="deliverables">Deliverables</h2>
<ul>
<li>
<p><strong>Project Idea</strong>, due Tuesday, 29 March, 4:59pm. Your project proposal (to be submitted using <a href="https://forms.gle/3tiqgYZRuNXrSzN56">this form</a> should include:</p>
<ol>
<li>The names and email ids for all members of your team.</li>
<li>Title of your project: short description that clearly captures your
project idea.</li>
<li>A short paragraph that describes the goal of your project.</li>
<li>A project jusfication that explains how your planned project is likely to satisfy at least three of the goals above (fun, relevant, technically interesting, useful).</li>
</ol>
</li>
<li>
<p><strong>Project Idea Presentation</strong> - in class on Wednesday, 30
March. Each team will have an opportunity to present your project
idea to the class with a single slide and few minutes.</p>
</li>
<li>
<p><strong>Project Proposal</strong>, due Friday, 8 April, 4:59pm. Your project proposal will include updates on the title, goal, and justification from your project idea, as well as:</p>
<ol start="5">
<li>State-of-the-art: your understanding of what is already available connected to the goal of your project. This could be other attempts to achieve the same goal (and why you think you can do something better), or work related to similar goals. A good state-of-the-art will include references to specific relevant work. For most types of projects, this would include research papers relevant to your topic.</li>
<li>A project plan that explains the main tasks needed to successfully complete your project and what you will actually do.</li>
<li>A management plan list of your team members and their roles and
responsibilities. If your team has more than two people, this should
also explain how you plan to coordinate and manage your team.</li>
</ol>
</li>
<li>
<p><strong>Project Update</strong>: due Tuesday, 19 April, 4:59pm. An update on how
your project is going, any changes from your original plans, and
summary of what progress you have been able to make.</p>
</li>
<li>
<p><strong>Project Presentations</strong>: in class on Wednesday, 27 April and
Monday, 2 May. Each team will give a brief presentation on their
project to the class.</p>
</li>
<li>
<p><strong>Project Reports or Artifacts</strong>: due Monday, 2 May, 9:59pm
(extensions possible if full team desires it and can agree on a
date that works well). Final submission of the results of your
project.</p>
</li>
</ul>
<p>Students who have an idea for their project early, and want more time to work on it, can replace Project 3 with additional time to work on their project by meeting this schedule:</p>
<ul>
<li>
<p><strong>Project Idea</strong>: submit by Tuesday, 15 March, 4:59pm (and get
approval from me, which should be later that day).</p>
</li>
<li>
<p><strong>Project Idea Presentation</strong>: in class on Wednesday, 30 March (same as everyone)</p>
</li>
<li>
<p><strong>Project Proposal</strong>: Friday, 1 April, 4:59pm.</p>
</li>
</ul>
<p>(The other deliverables will be on the same schedule, but expect to be
scheduled for the first in-class project presentation day on April
27.)</p>

      </div>
<hr class="post-separator"></hr>

    
    <h1><a href="/crisprpatent/">Update: CRISPR Patent Ruling</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-03-01 00:00:00 &#43;0000 UTC" itemprop="datePublished">1 March 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <p>USPTO has made a ruling in the CRISPR patent dispute we talked about
yesterday!</p>
<p>The ruling is in favor of MIT/Harvard (Broad Institute), determining
that they have priority in CRISPR-Cas9 system for editing eukaryotic
cells, over the CVC (this is the case name for the Berkeley/University
of Vienna assignees of the Doudna/Charpentier patent)
patent. (Although this ruling is now in effect, I doubt this well be
the end of things, and suspect the CVC lawyers are looking for ways to
appeal.)</p>
<p>You can read the full ruling here: <a href="/docs/usptocrispr.pdf">Decision on Priority
37 C.F.R. § 41.125(a)</a>. It is quite fascinating reading, with lots of quotes from emails between the scientists and scans from lab notebooks (see pages 30-31 for a nice sketch for tracrRNA and Cas9 cleaving).</p>
<p>Update: <a href="https://twitter.com/jsherkow/status/1498413788994940928">Jacob Sherkow&rsquo;s twitter
thread</a>
provides a great explanation of the whole episode and latest ruling
(and contrary my doubts above, thinks further appeals from CVC are
unlikely).</p>

      </div>
<hr class="post-separator"></hr>

    
    <h1><a href="/class12/">Class 12: CRISPR in Bacteria</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-02-28 00:00:00 &#43;0000 UTC" itemprop="datePublished">28 February 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <h2 id="readings">Readings</h2>
<p>Read before <strong>Wednesday, 2 March</strong>:</p>
<ul>
<li>DNA Book: <a href="https://berthub.eu/dna-book/private/bacterial-virus-warfare-crispr">Chapter 28: <em>Bacterial Virus Warfare</em></a> and <a href="https://berthub.eu/dna-book/private/genetic-manipulation-crispr">Chapter 29: <em>Genetic Manipulation, CRISPR &amp; beyond</em></a></li>
</ul>
<p>Read by the end of spring break:</p>
<ul>
<li>Prarthana Mohanraju, Kira S. Makarova, Bernd Zetsche, Feng Zhang,
Eugene V. Koonin, John van der Oost. <a href="/docs/mohanraju2016.pdf"><em>Diverse evolutionary roots and
mechanistic variations of the CRISPR-Cas systems</em></a>. Science, August 2016.</li>
</ul>
<h2 id="slides">Slides</h2>
<p>The slides from the 28 February class are here: <a href="https://www.dropbox.com/s/23ud9d46zsgjg4r/csbio-class12.pdf?dl=0">class12.pdf</a></p>
<h2 id="links">Links</h2>
<ul>
<li>
<p>Shigeru Iida, Markus B. Streiff, Thomas A. Bickle, and Werner Arber. <a href="/docs/iida1987.pdf"><em>Two DNA Antirestriction Systems Characterization of Bacteriophage P1, darA, and darB: Characterization of darA<sup>-</sup> Phages</em></a>. Virology, 1987.</p>
</li>
<li>
<p>Sydney Brenner, F. Jacob, M. Meselson. <a href="/docs/brenner1961.pdf"><em>An Unstable Intermediate Carrying Information from Genes to Ribosomes for Protein Synthesis</em></a>, Nature, May 1961.</p>
</li>
<li>
<p>Yoshizumi Ishino, Hideo Shinagawa, Kozo Makino, Mitsuko Amemura, and Atsuo Nakata. <a href="/docs/ishino1987.pdf"><em>Nucleotide Sequence of the <em>iap</em> Gene, Responsible for Alkaline Phosphatase Isozyme Conversion in Escherichia coli, and Identification of the Gene Product</em></a>, Journal of Bacteriology, December 1987. (This is the first paper to observe the palindromic repeats in bacterial DNA.)</p>
</li>
<li>
<p>Prarthana Mohanraju, Kira S. Makarova, Bernd Zetsche, Feng Zhang,
Eugene V. Koonin, John van der Oost. [<em>Diverse evolutionary roots and mechanistic variations of the CRISPR-Cas systems</em>]. Science, August 2016.</p>
</li>
<li>
<p>CRISPR patent: <a href="https://patents.google.com/patent/US20220010338A1/en"><em>Methods and compositions for RNA-directed target DNA modification and for RNA-directed transcription modulation</em></a></p>
</li>
</ul>

      </div>
<hr class="post-separator"></hr>

    
    <h1><a href="/class11/">Class 11: Editing Genomes</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-02-23 00:00:00 &#43;0000 UTC" itemprop="datePublished">23 February 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <h2 id="schedule-and-readings">Schedule and Readings</h2>
<p><strong>Thursday, 24 Feb</strong>:</p>
<ul>
<li><a href="/project2"><strong>Project 2</strong></a></li>
</ul>
<p>New readings for next week will be posted by Friday.</p>
<h2 id="slides">Slides</h2>
<p>The slides from the 21 February class are here: <a href="https://www.dropbox.com/s/px2yhuv9b70cguq/csbio-class11.pdf?dl=0">class11.pdf</a></p>
<h2 id="links">Links</h2>
<ul>
<li>
<p>Wladyslaw Altermann, Józef Kazmierczak. <a href="/docs/altermann2003.pdf"><em>Archean microfossils: a reappraisal of early life on Earth</em></a> (Describes fossils of life from over 3.4 Billion years ago.)</p>
</li>
<li>
<p>Anja Spang, Courtney W. Stairs, Nina Dombrowski, Laura Eme, Jonathan Lombard, Eva F. Caceres, Chris Greening, Brett J. Baker and Thijs J. G. Ettema. <a href="/docs/spang2019.pdf"><em>Proposal of the reverse flow model for the origin of the eukaryotic cell based on comparative analyses of Asgard archaeal metabolism</em></a>. Nature Microbiology, July 2019.</p>
</li>
<li>
<p><a href="https://www.ams.usda.gov/rules-regulations/be">Bioengineered Food Disclosure</a></p>
</li>
<li>
<p>The best recorded lectures I found on Recombinant DNA are MIT&rsquo;s 7.01 course (Eric Lander), starting with <a href="https://www.youtube.com/watch?v=uDXH6Uu0ghc"><em>Overview of Recombinant DNA, excerpt 1 | MIT 7.01SC Fundamentals of Biology</em></a>.</p>
</li>
<li>
<p>Pavan Umate, Narendra Tuteja and Renu Tuteja. <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3073292/pdf/cib0401_0118.pdf"><em>Genome-wide comprehensive analysis of human helicases</em></a>. Communicative and Integrative Biology, 2011.</p>
</li>
<li>
<p>Patent for recombinant DNA: <a href="https://patents.google.com/patent/US4237224A/"><em>Process for Producing Biologically Functional Molecular Chimeras</em></a> (Stanley Cohen and Herbert Boyer, 1974). Rajendra Bera, <a href="/docs/cohenboyerpatents.pdf"><em>The story of the Cohen-Boyer patents</em></a>, Current Science, March 2009.</p>
</li>
<li>
<p>Paul Berg, David Baltimore, Herbert W. Boyer, Stanley N. Cohen, Ronald W. Davis, David S. Hogness, Daniel Nathans, Richard Roblin, James D. Watson, Sherman Weissman, Norton D. Zinder. <a href="/docs/berg-biohazards1974.pdf"><em>Potential Biohazards of Recombinant DNA Molecules</em></a>. Science, 1974.</p>
</li>
<li>
<p><a href="https://futureoflife.org/2017/08/11/ai-principles/">Asilomar AI Principles</a> (2017).</p>
</li>
</ul>

      </div>
<hr class="post-separator"></hr>

    
    <h1><a href="/class10/">Class 10: Genomes and Phenotypes</a></h1>
<div class="post-metadata">
  <span class="post-date">
    <time datetime="2022-02-21 00:00:00 &#43;0000 UTC" itemprop="datePublished">21 February 2022</time>
  </span>
  
</div>

      <div class="post-body" itemprop="articleBody">
        <h2 id="schedule-and-readings">Schedule and Readings</h2>
<p><strong>Thursday, 24 Feb</strong>:</p>
<ul>
<li><a href="/project2"><strong>Project 2</strong></a></li>
</ul>
<h2 id="slides">Slides</h2>
<p>The slides from the 21 February class are here: <a href="https://www.dropbox.com/s/hywo3lj2k23caao/csbio-class10.pdf?dl=0">class10.pdf</a></p>
<h2 id="links">Links</h2>
<ul>
<li>
<p>Charles Darwin. <a href="https://www.google.com/books/edition/The_Variation_of_Animals_and_Plants_Unde/hoo5AAAAcAAJ?hl=en&amp;gbpv=1"><em>The Variation of Animals and Plants Under Domestication</em></a>. 1868. (Darwin&rsquo;s strange (to us today) theory on pangenesis is Chapter XXVII, starting on page 357.)</p>
</li>
<li>
<p>Gregor Mendel. <a href="https://www.biodiversitylibrary.org/page/48299076#page/133/mode/1up"><em>Versuche uber Pflanzen-Hybriden</em></a>. Proceedings of Natural History Society, 1866 <a href="http://www.esp.org/foundations/genetics/classical/gm-65.pdf">English Translation</a></p>
</li>
<li>
<p>R. A. Fisher. <a href="/docs/fisher1936.pdf"><em>Has Mendel&rsquo;s Work Been Rediscovered?</em></a>, 15 April 1936. (This isn&rsquo;t the paper we talked about in class, but is an interesting later one.)</p>
</li>
<li>
<p>Matti Pirinen&rsquo;s GWAS course: <a href="https://www.mv.helsinki.fi/home/mjxpirin/GWAS_course/">Genome-wide Association Studies</a>, University of Helsinki.</p>
</li>
<li>
<p>Robert J. Klein, Caroline Zeiss, Emily Y. Chew, Jen-Yue Tsai, Richard S. Sackler, Chad Haynes, Alice K. Henning, John Paul SanGiovanni, Shrikant M. Mane, Susan T. Mayne, Michael B. Bracken, Frederick L. Ferris, Jurg Ott, Colin Barnstable, Josephine Hoh. <a href="/docs/klein2005.pdf"><em>Complement Factor H Polymorphism in Age-Related Macular Degeneration</em></a>. Science, 15 April 2005.</p>
</li>
<li>
<p>James J. Lee, et al. <a href="/docs/gwaseducation.pdf"><em>Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals</em></a>. Nature Genetics, August 2018.</p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/gwas/diagram">GWAS Catalog</a></p>
</li>
</ul>

      </div>
<hr class="post-separator"></hr>

    

   

    <footer>
      <nav>
	<a href="/post/" class="button hollow primary">All Posts</a>
      </nav>
    </footer>

</div>
</div>


    </main>
    
    
    <footer class="whatisthis">
  <hr />
  <div class="row">
    <div class="column small-12 medium-6">
      <a href="//computingbiology.github.io"><b>cs 4501: Computational Biology / Biological Computing</b></a><br>
      <a href="//www.cs.virginia.edu">University of Virginia</a>
    </div>
    <div class="column small-14 medium-5">
      <font size="-1">
      Subscribe to
	the <a href="/index.xml"><i class="fa fa-rss-square"></i>&nbsp;RSS feed</a>.
      <a id="searchsite">
	<form method="get" action="https://duckduckgo.com/">
	  <label for="search-field" class="show-for-sr">Search with DuckDuckGo</label>
	  <input type="search" name="q" maxlength="255" placeholder="Search with DuckDuckGo" id="search-field">
	  <input type="hidden" name="sites" value="https://computingbiology.github.io/"/>
	  <input type="hidden" name="k7" value="#faf8f8"/>
	  <input type="hidden" name="kj" value="#b33"/>
	  <input type="hidden" name="ky" value="#fafafa"/>
	  <input type="hidden" name="kx" value="b"/>
	  <input type="hidden" name="ko" value="-1"/>
	  <input type="hidden" name="k1" value="-1"/>
	  <input type="submit" value="DuckDuckGo Search" style="visibility: hidden;" />
	</form>
      </a>
</font>
    </div>
  </div>
  </hr>

</footer>


    
    
    <div class="endofpage">
    </div>

    <script src="/js/jquery.js"></script>
    <script src="/js/what-input.js"></script>
    <script src="/js/foundation.min.js"></script>
    <script src="/js/finite.js"></script>

    
    <script src="/js/highlight.pack.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>

    
    
    
  </body>
</html>
